index.bs

```diff
@@ -313,7 +313,8 @@ directory rather than a file or files. When specified, the behavior on
 the selection of a directory is as if all files with that directory as
 an ancestor were selected. In addition, the
 {{File/webkitRelativePath}} property of each {{File}} is set to a
-[=relative path=] from the selected directory to the file.
+[=relative path=] starting from (and including) the selected directory
```

Comment:
A relative path from selected "/" or "C:\\" might be unclear. If I follow this definition literally, a resultant relative path starts with "/" or "C:\\".  I haven't checked browsers' behavior in this case.

Reply:
I confirmed Edge and Firefox produced webkitRelativePath starting with "D:/" if I selected "D:/". So this definition is ok.
I couldn't check Safari behavior due to OOM.

Reply:
Thanks for checking those edge cases!